src/main/java/org/continuouspoker/dealer/persistence/daos/GameDAO.java

```diff
@@ -0,0 +1,78 @@
+package org.continuouspoker.dealer.persistence.daos;
+
+import java.time.Instant;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+
+import io.quarkus.hibernate.orm.panache.PanacheEntityBase;
+import io.quarkus.hibernate.orm.panache.PanacheQuery;
+import io.quarkus.panache.common.Sort;
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.transaction.Transactional;
+import org.continuouspoker.dealer.Team;
+import org.continuouspoker.dealer.game.Game;
+import org.continuouspoker.dealer.persistence.entities.GameBE;
+import org.continuouspoker.dealer.persistence.entities.ScoreRecordBE;
+import org.continuouspoker.dealer.persistence.entities.TeamBE;
+import org.continuouspoker.dealer.persistence.entities.TeamScoreRecordBE;
+import org.continuouspoker.dealer.persistence.mappers.GameMapper;
+import org.continuouspoker.dealer.persistence.mappers.TeamMapper;
+import org.mapstruct.factory.Mappers;
+
+@ApplicationScoped
+public class GameDAO {
+    private static final int SCORE_LIMIT = 100;
+    TeamMapper teamMapper = Mappers.getMapper(TeamMapper.class);
+    GameMapper gameMapper = Mappers.getMapper(GameMapper.class);
```

Comment:
Variable scope seems too public here. Also, I think that Mappers can be used as beans to be injected here.